itou/www/news/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from itou.www.news import views
+
+
+app_name = "news"
+
+
+urlpatterns = [
+    path("", views.NewsView.as_view(), name="home"),
```

Comment:
Pour coller aux modèles ?
```suggestion
    path("", views.AnnouncementsView.as_view(), name="home"),
```